README.md

```diff
@@ -70,8 +70,8 @@ user_list is the list of local scores
 
 **Request :**
 ```
-curl base_url/user/create/ \
-    -X POST \
+curl base_url/users/ \
```

Comment:
@omkarjc27 we're going to start keeping tack of the api endpoints and their details in the docs directory. Take a look at my PR and introduce your changes for the interface there.

Reply:
I suggest we delete the README.md file in the root directory and keep the all documentation in docs/src/index.html.md . So there will be no need to update 2 docs when changes are made to the API.

Reply:
We can keep readme.md but remove duplicate info. A couple thing we can update it with 
- Setting up and running dev environment
- Link to docs page